src/coreclr/src/System.Private.CoreLib/src/System/Reflection/CustomAttribute.cs

```diff
@@ -1240,35 +1641,31 @@ internal static bool IsAttributeDefined(RuntimeModule decoratedModule, int decor
                     {
                         // Metadata is always written in little-endian format. Must account for this on
                         // big-endian platforms.
+                        int data = Unsafe.ReadUnaligned<int>((void*)blobStart);
```

Comment:
This is not correct change wrt. endianness.

Reply:
Fixed